firefox_puppeteer/ui/windows.py

```diff
@@ -188,7 +188,7 @@ def __eq__(self, other):
 
     @property
     def closed(self):
-        """Returns closed state of the chrome window
+        """Returns closed state of the chrome window.
```

Comment:
I don't see any sentences or verb phrases there lacking periods, and in other places in the file, noun phrases don't use periods, so I tried to make those consistent.